automated-testing/performance-testing/readme.md

```diff
@@ -1,67 +1,357 @@
+    load. Poor performing applications require more hardware and thus are more
+    expensive to operate. Capacity planning involves identifying business
+    expectations, the periodic fluctuations of application usage, considering
+    the cost of running the hardware and software infrastructure.
+
+- To reduce **downtime**
+
+  - Systems that go down cost money. For every second thought. Amazon Web
+    services is down, is a loss of eleven hundred dollars per second. And for
+    financial services firms’ downtime could mean lawsuits and customer
+    attrition.
 
 ## Key Performance Testing categories
 
-<!-- markdownlint-disable no-duplicate-heading -->
-### Performance Testing
-<!-- markdownlint-enable no-duplicate-heading -->
+Performance testing is a broad topic. There are many areas where you can perform
+tests. In broad strokes you can perform tests on the backend and on the front
+end. You can test the performance of individual components as well as testing
+the end\-to\-end functionality.
+
+There are several categories of tests as well:
 
-  This category is the super set of all sub categories of performance related testing. It validates/determines the speed, scalability or reliability characteristics of the system under test. Performance testing focuses on achieving the response times, throughput, and resource utilization levels which meet the performance objectives of a system
+### Load Testing
 
-### **[Load Testing](./load-testing.md)**
-  
-  This is the subcategory of performance testing which focuses on validating the performance characteristics of a system, when the system faces load volumes which are expected during production operation. **Endurance Test** or **Soak Test** is a load test carried over a long duration ranging from several hours to days.
+This is the subcategory of performance testing which focuses on validating the
+performance characteristics of a system, when the system faces load volumes
+which are expected during production operation. **Endurance Test** or **Soak
+Test** is a load test carried over a long duration ranging from several hours to
+days.
 
 ### Stress Testing
 
-  This is the subcategory of performance testing which focuses on validating the performance characteristics of a system when the system faces extreme load. The goal is to evaluate how does the system handles being pressured to its limits, does it recover (i.e. scale-out) or does it just break and fail?
+This is the subcategory of performance testing which focuses on validating the
+performance characteristics of a system when the system faces extreme load. The
+goal is to evaluate how does the system handles being pressured to its limits,
+does it recover (i.e., scale\-out) or does it just break and fail?
+
+### Endurance testing
+
+The goal of endurance testing is to make sure that the software can maintain
+good performance when extended periods of load.
```

Comment:
suggestion: good performance under extended periods of load

Reply:
Agreed. More clear. Easier to read.

Reply:
Sorry, I think I did not elaborate. I meant to change the text in the paragraph "can maintain
good performance when extended periods of load." to "can maintain
good performance under extended periods of load."...I think the title should remain "Endurance testing" as that is the commonly known name for this kind of test